samples/zigbee/light_bulb/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _zigbee_light_bulb_sample:
+
+Zigbee: Light bulb
+##################
+
+This Zigbee light bulb sample demonstrates a simple light bulb whose brightness can be regulated by a different device.
+
+You can use this sample together with :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` and Zigbee light switch to set up a basic Zigbee network.
```

Comment:
```suggestion
You can use this sample together with the :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` and Zigbee light switch to set up a basic Zigbee network.
```
I guess the light switch sample will be added later? Should this be mentioned here?

Reply:
No, light switch is in a PR that will be reviewed and added after this one is merged.

Reply:
Are we 100% sure that the other PR will go in right after the other one? Otherwise, I would add a "(to be added)" here just to make this clear - and the file needs to be updated anyway, so it's no big deal to remove that comment again.

Reply:
Yes, it will go in right after.